web_invalid_tab/static/src/js/view_form.js

```diff
@@ -0,0 +1,62 @@
+openerp.web_invalid_tab = function(instance) {
+
+    var tab_selector = 'div[role="tabpanel"]';
+
+    function tab_link(tab) {
+        return $("a[href='#" + tab.attr('id') + "']");
+    }
+
+    function is_visible(tab, element) {
+        if ($(element).hasClass('oe_form_invisible') || element.style.display == 'none') {
```

Comment:
@hparfr yes is more expressive, but if a required field is inside a tab which is not active then I will get that the field is hidden. Then I can not use ':hidden' or ':visible' because I need to know if the field is visible when the tab is active. That's the reason why I created the function is_visible. Do you have any other idea?